CondFormats/L1TObjects/src/L1TMuonBarrelKalmanParams.cc

```diff
@@ -0,0 +1,11 @@
+#include "CondFormats/L1TObjects/interface/L1TMuonBarrelKalmanParams.h"
+
+L1TMuonBarrelKalmanParams::L1TMuonBarrelKalmanParams()
+    : pnodes_(NUM_BMTF_PARAM_NODES)  //, l1mudttfmasks_(1)
+{
+  version_ = Version;
+  pnodes_[CONFIG].type_ = "unspecified";
+  pnodes_[CONFIG].fwVersion_ = 0;  //default to recognize a RCD that is not filled
+  // pnodes_[CONFIG].sparams_.clear();
```

Comment:
Please remove the commented out code if not needed, or add a line which explains why it must stay there

Reply:
removed can be reimplemented if needed